go-controller/pkg/util/policy.go

```diff
@@ -0,0 +1,85 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+// This file contains the util functions commonly used by
+// network policy and admin network policy controllers
+// NOTE: The pieces that are ACL utils must go into libovsdb/utils/acl.go file instead
+// This is only for Policy utils
+
+// ACL L4 Match Construct Utils
+const (
+	// emptyProtocol is used to create ACL for gressPolicy that doesn't have port policies hence no protocols
+	EmptyProtocol = "None"
+)
+
+// GetPortProtocol returns the OVN syntax-specific protocol value for a v1.Protocol K8s type
+func GetPortProtocol(proto v1.Protocol) string {
+	var protocol string
+	switch proto {
+	case v1.ProtocolTCP:
+		protocol = "tcp"
+	case v1.ProtocolSCTP:
+		protocol = "sctp"
+	case v1.ProtocolUDP:
+		protocol = "udp"
+	}
+	return protocol
+}
+
+type PolicyPort struct {
```

Comment:
I would use NetworkPolicyPort as name.

I would probably avoid the `GetPortProtocol` method above, and just have
```
func GetNetworkPolicyPort(networkPolicyPort v1.NetworkPolicyPort) NetworkPolicyPort
```

Reply:
Another naming option is to focus it to OVN. 

So this type would be named like `AclL4Match`

And you would have
```
func GetAclL4MatchFromPolicy(networkPolicyPort v1.NetworkPolicyPort) AclL4Match
```
and
```
func GetAclL4MatchesAsString( matches []AclL4Match) []string
```

Reply:
I prefer your first suggestion here.. going with that but let's see if its ok in iteration2